lib/validation/rules/after.ts

```diff
@@ -0,0 +1,15 @@
+import { createRule } from '../Rule'
+import { after as baseAfter } from '../validators/after'
+
+export const message = {
+  en: 'The date must be after the specified date.',
```

Comment:
Can we have actual date value shown in message here? I think we can access the specified date like this 👀  We do this in things like `maxValue` validation.

```ts
export const message = {
  en: (max: string) => `The value must be less than or equal to ${max}.`,
  ja: (max: string) => `この値は最大${max}です。`
}

export function maxValue(max: number, msg?: string) {
  return createRule({
    message: ({ lang }) => msg ?? message[lang](format(max)),
    optional: true,
    validation: (value) => baseMaxValue(value, max)
  })
}
```

Reply:
Updated. Please check the Japanese translations once.

Reply:
Thanks! I'll adjust the JA message and merge it 👍